src/lib/viewers/media/DashViewer.js

```diff
@@ -27,9 +27,10 @@ class DashViewer extends VideoBaseViewer {
         this.switchHistory = [];
 
         // tracks
-        this.hdRepresentation = {};
-        this.sdRepresentation = {};
+        this.hdVideoId = -1;
+        this.sdVideoId = -1;
```

Comment:
We would normally do `null` or just initialize it as `undefined` but if -1 makes more sense since these are numeric IDs I'm ok with that. What are other peoples thoughts on this?